### ISSUE TITLE
unable to render boundary/admin layer

### ISSUE THREAD

**Issue by mkandulavm** [tangrams/tangram-es#1999]
Hi

I have followed this article to extact MVT files from OSM PBF files.
https://blog.jawg.io/how-to-make-mvt-with-postgis/

In their example using the browser mapbox renderer, the buildings (polygon), ameneties (POI) and admin (linestring)  are visible.

I am using the same MVT tiles and trying to view with tangram-es.
I have attached the scene file I am using.
[scene.zip](https://github.com/tangrams/tangram-es/files/2759478/scene.zip)

I modified the layer name in the scene file from pois -> ameneties and boundaries -> admin to match with the data in tiles.

In tangram-es, I am able to see the buildings (polygon) and ameneties (POI). However the admin (linestring) is not visible. Can someone please help me identify what I need to modify in the scene file so the admin layer is visible ?

Thanks.

**Comment by matteblair** [comment]
Hi @mkandulavm! I think I understand what you're doing here, this is a cool idea. The scene file you attached doesn't have any obvious problems that I can see. Can you see if there is any error output in the console from Tangram ES?

**Comment by mkandulavm** [comment]
Thanks for the reply @matteblair  ! 
I could not find anything useful in the log. I am attaching it. There is only tile not found error, but those are for tiles that are not generated.

Is there a more detailed or verbose flag for logging ?

[tangram-log.txt](https://github.com/tangrams/tangram-es/files/2762788/tangram-log.txt)

Could you give me some suggestions to explore how to find the problem ?
Basically it looks like I missing something to map in the scene file to see the admin linestrings or something by default is overriding the display (zoom, filter?).

**Comment by mkandulavm** [comment]
HI @matteblair , any help please ?

**Comment by matteblair** [comment]
Hey, thanks for your patience with this! I don't see anything suspicious in the logs, now I'm wondering if the issue is something about the tile data instead. Do you have sample data containing boundaries that you can share?

**Comment by mkandulavm** [comment]
Hi @matteblair 

I am attaching sample tiles at zoom 15 and 16 for a small region of paris.
[tiles.zip](https://github.com/tangrams/tangram-es/files/2781353/tiles.zip)

**Comment by mkandulavm** [comment]
HI @matteblair 

Could you suggest some debugging tips or where in the code I can start probing?  :)
